### PR TITLE
[CDAP-16263] Fix log messages with `null` object argument is getting dropped by the logging system

### PR DESCRIPTION
**Pull request by tivv**

### CODE REVIEW

File: cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java [hunk 1]
```diff
@@ -20,11 +20,13 @@
...
 import ch.qos.logback.classic.spi.IThrowableProxy;
 import ch.qos.logback.classic.spi.LoggerContextVO;
-import com.google.common.base.Objects;
 import io.cdap.cdap.common.logging.LoggingContext;
 import org.slf4j.Marker;
 
 import java.util.Map;
+import java.util.Objects;
+
+import static com.google.common.base.Objects.toStringHelper;
```
Comment (chtyim) [inline cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java]: Don't use static import.

File: cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java [hunk 2]
```diff
@@ -123,14 +125,14 @@ public void prepareForDeferredProcessing() {
     if (args != null && argumentArray == null) {
       argumentArray = new Object[args.length];
       for (int i = 0; i < args.length; i++) {
-        argumentArray[i] = args[i].toString();
+        argumentArray[i] = Objects.toString(args[i]);
```
Comment (chtyim) [inline cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java]: Prefer to use the `java.util.Objects` instead.

Comment (chtyim) [reply cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java]: This is a good catch

Comment (tivv) [reply cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java]: Sure, changed to fully qualified import

**Comment by tivv** [comment]
@chtyim changed to fully qualified import and removed the static one, please review.

**Review by chtyim — APPROVED** [review]

**Pull request merged**